Python/Setup/swix/Workloads/Microsoft.VisualStudio.Workload.Python.swixproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="SetupProjectBefore.settings" />
+
+  <ItemGroup>
+    <Package Include="$(MSBuildProjectName).res.swr" />
+
+    <Dependency Include="Microsoft.PythonTools" />
+    <Dependency Include="Microsoft.PythonTools.Web" />
+    <Dependency Include="Microsoft.PythonTools.Samples">
+      <Type>Optional</Type>
+    </Dependency>
+    <Dependency Include="Microsoft.PythonTools.Uwp">
```

Comment:
If we can't get away without them, sure. Web tools we might want to defer until someone creates a web project and then prompt, it really depends how big they are. Or they'll be optional+default

---
In reply to: [69645747](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69645747) [](ancestors = 69645747)